fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -182,7 +182,11 @@ const RollCallOpen = ({
           <Text style={Typography.paragraph}>{STRINGS.roll_call_open_attendee}</Text>
           <View>
             <QRCode
-              value={ScannablePopToken.encodePopToken({ pop_token: popToken })}
+              value={
+                popToken !== ''
+                  ? ScannablePopToken.encodePopToken({ pop_token: popToken })
+                  : undefined
```

Comment:
although I don't see how, this technically could throw.
Would it make sense to do this operation beforehand and wrap it like you did in digital cash ?

Reply:
this could throw since the popToken is initially empty, and it is changed through a promise (`generateToken`), so for a brief moment the popToken will be an empty string
I will wrap it earlier 👍